File: matrix multiplication.c
//matrix multiplication //
#include<stdio.h>
main()
{
	int a1,a2,a3;
	printf ("enter a,b,c(m[a][b]&m[b][c]):\n");
	scanf("%d%d%d",&a1,&a2,&a3);
	
	int m1[a1][a2],m2[a2][a3],c[a1][a3];
		printf ("enter values of m[%d][%d]\n",a1,a2);
	for(int i=1;i<=a1;i++)
	{
		for(int j=1;j<=a2;j++)
		scanf("%d",&m1[i][j]);
	}
		printf ("\n enter values  of m[%d][%d]\n",a2,a3);
			for(int i=1;i<=a2;i++)
	{
		for(int j=1;j<=a3;j++)
		scanf("%d",&m2[i][j]);
	}
	printf("MULTIPLICATION OF MATRIX :\n");
		for(int i=1;i<=a1;i++)
	{ 
		for(int j=1;j<=a3;j++)
		{   int x=0;
		    for(int k=1;k<=a2;k++)
		    x=x+(m1[i][k]*m2[k][j]);
			c[i][j]=x;
		}
	}
		for(int i=1;i<=a1;i++)
	{
		for(int j=1;j<=a3;j++)
		printf("%d\t",c[i][j]);
		printf("\n");
	}   
}

File: Python/MatrixMultiplication.py
# A Program for multiplication of two matrices of any order

# Let first matrix is a 3x4 matrix
A = [[10,4,6,12],
    [9,3,7,1],
    [12,8,2,5]]

# Second matrix is a 4x3 matrix
B = [[5,2,4],
    [9,1,7],
    [3,8,6],
    [6,1,8]]


# So Resultant matrix will be a 3x3 matrix
result = [[0,0,0],
         [0,0,0],
         [0,0,0]]

# Matrix multiplication
for m in range(len(A)):
   for n in range(len(B[0])):
       for o in range(len(B)):
           result[m][n] += A[m][o] * B[o][n]

for r in result:
   print(r)
